package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "knockout.datepicker",
+  "version": "3.0.0",
+  "description": "Simple knockout binding which wraps jQuery UI datepicker",
+  "main": "knockout.datepicker.js",
+  "scripts": {
+    "lint": "eslint . --format=unix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/civicsource/knockout.datepicker.git"
+  },
+  "keywords": [
+    "knockout",
+    "datepicker",
+    "jquery-ui"
+  ],
+  "author": "CivicSource",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/civicsource/knockout.datepicker/issues"
+  },
+  "homepage": "https://github.com/civicsource/knockout.datepicker#readme",
+  "dependencies": {
+    "jquery": "*",
```

Comment:
you should take a peer dependency on knockout in all these knockout plugins. I would use knockout 3.x